pom.xml

```diff
@@ -11,6 +11,14 @@
     <properties>
         <graphdb.version>9.8.0</graphdb.version>
         <dependency.check.version>6.2.2</dependency.check.version>
+
+        <internal.repo>http://maven.ontotext.com/content/repositories/owlim-releases</internal.repo>
+        <snapshots.repo>http://maven.ontotext.com/content/repositories/owlim-snapshots</snapshots.repo>
+
+        <!-- Empty defaults to keep things happy, argLine can be overridden on command line,
```

Comment:
No need to have these two properties - they aren't used in the pom so better remove them to avoid confusion.